bzip2/tests/test.bats

```diff
@@ -0,0 +1,5 @@
+@test "Version matches plan" {
+  TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+  actual_version="$(hab pkg exec "${TEST_PKG_IDENT}" bzip2 --help 2>&1 | grep 'bzip2, a block-sorting file compressor' | grep -oP '(?<=Version )([^,]*)')"
```

Comment:
This test is dependant on the wording of the help command not changing.
Is it possible to just use the second `grep` and only depend on `Version` being in the text?

Reply:
Hi @james-stocks.  Since I'm going to be closing this PR as a duplicate of #2741, I'm fixing this in the other PR.